client/src/components/Layout/NavBar/index.jsx

```diff
@@ -1,30 +1,40 @@
 import React from 'react';
 import { useStateValue } from 'react-conflux';
 import styled from 'styled-components';
+import { BaseButton } from '../../../styles/themes';
 
 import { useAuth0 } from '../../../auth/authWrapper';
 import { globalContext } from '../../../store/reducers/globalReducer';
 
-function NavBar() {
+function NavBar(props) {
```

Comment:
Scaffolding out the `NavBar` - This will make it easier later to implement the final edition